spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/rsocket/RSocketProperties.java

```diff
@@ -121,6 +123,74 @@ public void setSsl(Ssl ssl) {
 			this.ssl = ssl;
 		}
 
+		public Spec getSpec() {
+			return this.spec;
+		}
+
+		public void setSpec(Spec spec) {
+			this.spec = spec;
+		}
+
+		public static class Spec {
+
+			/**
+			 * Sub-protocol to use in websocket handshake signature. Null by default.
```

Comment:
There's no need to write "by default" in the description as we harvest defaults and process them separately.